frontend/packages/console-plugin-sdk/src/typings/dashboards.ts

```diff
@@ -31,6 +33,25 @@ namespace ExtensionProperties {
     /** The Prometheus query */
     query: string;
   }
+
+  export interface DashboardsTab {
+    /** The tab's ID which will be used as href within dashboards page */
+    id: string;
+
+    /** The tab title */
+    title: string;
+  }
+
+  export interface DashboardsCard {
+    /** The tab's ID where this card should be rendered */
+    tabID: string;
```

Comment:
Can we name this `tab` ? (Similar to `NavItem.perspective` referring to a perspective ID.)

Reply:
sure, renamed to `tab`